web-console/src/utils/general.tsx

```diff
@@ -66,13 +66,11 @@ export function makeBooleanFilter(): FilterRender {
   };
 }
 
-export function customTableFilter(filter: Filter, row?: any): boolean | string {
-  const id = filter.pivotId || filter.id;
-  const clientSideFilter: boolean = row !== undefined;
-  if (clientSideFilter && row[id] === undefined) {
+export function customTableFilter(filter: Filter, clientSideFilter: boolean, value?: any): boolean | string {
```

Comment:
This function has no clear meaning, could you please refactor this into two functions one for `clientSideFilter` `true` and one for `false`